masterlist.yaml

```
@@ -18600,3 +18600,12 @@ plugins:
       - Invent.Remove
       - Names
       - Relev
+
+# Surival Mode Patches - aMidianBorn Content Add-on
+  - name: 'Survival Mode Patches - AMB Content Addon.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19154'
+        name: 'Survival Mode Patch Collection'
+    msg:
+      - <<: *obsolete
+        subs: [ '[aMidianBorn_ContentAddon.esp](https://www.nexusmods.com/skyrimspecialedition/mods/35390)' ]
```

I would suggest changing it from the `Obsolete` alias to `alreadyInOrFixedByX`.
It will prevent anyone using the patch with the new version, and not give unnecessary messages to anyone with the old version of AMB addon.

```suggestion
    msg:
      - <<: *alreadyInOrFixedByX
        subs: [ 'aMidianBorn Content Addon SE' ]
        condition: 'active("aMidianBorn_ContentAddon.esp") and version("aMidianBorn_ContentAddon.esp", "2.0", >=)'
```